torch/csrc/jit/codegen/cuda/expr_simplifier.cpp

```diff
@@ -480,14 +536,100 @@ Val* unflatten(Val* value, const std::list<ValInfo>& variables) {
     }
     return lhs;
   }
+  return value;
+}
+
+Val* unflatten(Val* value, const std::list<ValInfo>& variables) {
+  using namespace std::placeholders;
+  return recurseDown(value, std::bind(unflattenRule, _1, variables));
```

Comment:
nit: can this be written like: `[&variables](Val* val) { return unflattenRule(val, variables); }`? I always think odd to have a variable like `_1`.

Reply:
changed